db/migrate/20241224142141_add_total_paid_amount_cents_to_invoices.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class AddTotalPaidAmountCentsToInvoices < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    add_column :invoices, :total_paid_amount_cents, :bigint, null: true, default: 0
+    # Backfill
+    Invoice.in_batches(of: 10_000).each do |batch|
+      batch.update_all(total_paid_amount_cents: 0) # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Do we need this if default is zero?

Reply:
i should be without the default to not lock the db I will remove it